packages/headless/src/features/search/search-actions.ts

```diff
@@ -50,10 +50,6 @@ export interface SearchAction {
   actionCause: string;
 }
 
-export const nullActionCause = (): SearchAction => ({
-  actionCause: null as unknown as string,
-});
-
```

Comment:
I realized that instead of sending null we can simply not send any actionCause by doing `executeSearch({legacy:...}) `only and not `executeSearch({legacy:..., next:...})`